linkerd-await.yaml

```diff
@@ -25,8 +25,8 @@ pipeline:
   - runs: |
       cargo fetch
       cargo auditable build --frozen --release
-      mkdir -p ${{targets.destdir}}/usr/bin
-      mv target/release/linkerd-await ${{targets.destdir}}/usr/bin
+      mkdir -p ${{targets.destdir}}/usr/lib/linkerd
+      mv target/release/linkerd-await ${{targets.destdir}}/usr/lib/linkerd
```

Comment:
these should be a sub-package compats 
@xnox 
```suggestion
      mkdir -p ${{targets.destdir}}/usr/lib/linkerd
      mv target/release/linkerd-await ${{targets.destdir}}/usr/lib/linkerd
```

Reply:
@xnox, FYI - can you ack / approve here?

Reply:
`/usr/lib/linkerd` is valid without need for introducing a compat package (see `systemd`, `cups`, `firewalld` and others). I'd prefer upstream had used `libexec`, but it's still an acceptable path